stages/org.osbuild.oscap.remediation

```diff
@@ -89,34 +86,40 @@ SCHEMA = """
 }
 """
 
-# Map containing the translations between
-# our configuration names and the OpenScap
-# configuration names.
-OSCAP_OPTIONS = {
-    "datastream" : "OSCAP_REMEDIATE_DS",
-    "profile_id" : "OSCAP_REMEDIATE_PROFILE_ID",
-    "datastream_id" : "OSCAP_REMEDIATE_DATASTREAM_ID",
-    "xccdf_id" : "OSCAP_REMEDIATE_XCCDF_ID",
-    "benchmark_id" : "OSCAP_REMEDIATE_BENCHMARK_ID",
-    "tailoring" : "OSCAP_REMEDIATE_TAILORING",
-    "tailoring_id" : "OSCAP_REMEDIATE_TAILORING_ID",
-    "arf_result" : "OSCAP_REMEDIATE_ARF_RESULT",
-    "html_report" : "OSCAP_REMEDIATE_HTML_REPORT",
-    "verbose_log" : "OSCAP_REMEDIATE_VERBOSE_LOG",
-    "verbose_level" : "OSCAP_REMEDIATE_VERBOSE_LEVEL",
-}
+XML_RESULTS = "eval_remediate_results.xml"
+HTML_REPORT = "eval_remediate_report.html"
 
 def main(tree, options):
-    filename = options["filename"]
     config = options["config"]
-
-    with open(f"{tree}/{filename}", "w", encoding="utf-8") as f:
-        for ours, theirs in OSCAP_OPTIONS.items():
-            value = config.get(ours)
-            if value:
-                f.write(f"{theirs}={value}\n")
-
-    os.symlink(f"{filename}", f"{tree}/system-update")
+    datastream = config.get("datastream")
+    profile = config.get("profile_id")
+    ds_id = config.get("datastream_id")
+    xccdf_id = config.get("xccdf_id")
+    xml_results = config.get("xml_results", XML_RESULTS)
+    html_report = config.get("html_report", HTML_REPORT)
+
+    os.mkdir(f"{tree}/openscap_data")
```

Comment:
I assume the requirement to put this on the top level directory comes from openscap, because it is a bit iffy.

Reply:
Ah I don't think it's a requirement, that's where they are doing it in anaconda.
Where would you suggest we put it?

Reply:
I'm just making some changes here. I think it might be better to make the path variable optional and only save the results if the path is specified.